getpaid/backends/transferuj/views.py

```diff
@@ -45,6 +45,9 @@ class SuccessView(DetailView):
     def render_to_response(self, context, **response_kwargs):
         return HttpResponseRedirect(reverse('getpaid-success-fallback', kwargs={'pk': self.object.pk}))
 
+    def post(self, *args, **kwargs):
```

Comment:
Well I don't know about this one... This isn't wrong but it's just the `DetailView` is for get only. So I though perhaps it would be better to change the base class and not use `DetailView` here. Maybe I'm just being too picky... and it  doesn't matter.